Library/include/CSP/Systems/Spaces/Space.h

```diff
@@ -328,15 +331,18 @@ class CSP_API SpacesMetadataResult : public csp::systems::ResultBase
 
 public:
 	const csp::common::Map<csp::common::String, csp::common::Map<csp::common::String, csp::common::String>>& GetMetadata() const;
+	const csp::common::Map<csp::common::String, csp::common::Array<csp::common::String>>& GetTags() const;
 
 private:
 	SpacesMetadataResult(void*) {};
 	SpacesMetadataResult(csp::systems::EResultCode ResCode, uint16_t HttpResCode) : csp::systems::ResultBase(ResCode, HttpResCode) {};
 	SpacesMetadataResult() {};
 
 	void SetMetadata(const csp::common::Map<csp::common::String, csp::common::Map<csp::common::String, csp::common::String>>& InMetadata);
+	void SetTags(const csp::common::Map<csp::common::String, csp::common::Array<csp::common::String>>& InTags);
 
 	csp::common::Map<csp::common::String, csp::common::Map<csp::common::String, csp::common::String>> Metadata;
+	csp::common::Map<csp::common::String, csp::common::Array<csp::common::String>> Tags;
```

Comment:
[Question] The tags aren't already stored in the `Metadata` object are they? I just wanna be sure we're not duplicating data and giving it a chance to get out of sync.
(I don't _think_ they are, I had a quick check of the call path.)

Reply:
I don't think they are, but I would also prefer someone who is more familiar with the code to confirm

Reply:
I'm not sure that person exists, you should check in the debugger if there's any doubt.

Reply:
I had checked the debugger, they're not in there. you can see it in the test as well. however, it would help to know why there was never a getter for tags, and why we have 2 things (tags on one hand and metadata on the other) in there. having two separate things in something called SpaceMetadata feels wrong. but then again, they are updated at the same time in UpdateSpaceMetadata as well, so it's not new. I don't know what the metadata is as opposed to tags and what their relation to each other is (could tags be stored as metadata under a "tags" entry?), so it would help to talk to someone who wrote this or read a design document. I'll have a look tomorrow

Reply:
found it. OF-1455

Reply:
I see yeah, looks to be a different field. Thanks for confirming.